src/calculateRentalCost.js

```diff
@@ -26,9 +26,21 @@
  *
  * @return {number}
  */
+const longTerm = 7;
+const longTermDiscount = 50;
+const shortTerm = 3;
+const shortTermDiscount = 20;
 
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = days * 40;
+
+  if (days >= longTerm) {
+    return basePrice - longTermDiscount;
+  } else if (days >= shortTerm) {
```

Comment:
try to make it worki without `else if` - [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) point 4, 5